google/cloud/CMakeLists.txt

```diff
@@ -28,7 +28,11 @@ set(DOXYGEN_PROJECT_BRIEF "C++ Client Library for Google Cloud Platform")
 set(DOXYGEN_PROJECT_NUMBER
     "${GOOGLE_CLOUD_CPP_VERSION_MAJOR}.${GOOGLE_CLOUD_CPP_VERSION_MINOR}.${GOOGLE_CLOUD_CPP_VERSION_PATCH}")
 set(DOXYGEN_PREDEFINED "GOOGLE_CLOUD_CPP_NS=v${GOOGLE_CLOUD_CPP_VERSION_MAJOR}")
-set(DOXYGEN_EXCLUDE_PATTERNS "internal/*;*_test.cc;*/*_test.cc")
+set(DOXYGEN_EXCLUDE_PATTERNS
+        "*/google/cloud/internal/*"
+        "*/google/cloud/bigtable/*"
```

Comment:
Previously, we just excluded docs for `internal` and test files; now we're excluding docs for Bigtable and Firestore altogether. Should this only exclude `bigtable/internal`, `bigtable/tests` and similarly for Firestore? Otherwise, I'm afraid we might not generate any docs for Bigtable and Firestore at all.

Reply:
Before this change we generated 4 Doxygen "directories" (pages?):

- `bigtable/*`
- `google/cloud/*`
- `google/cloud/firestore/*`
- `storage/*`

By accident `google/cloud/*` included the `google/cloud/firestore/*` documentation.  I have a separate bug (#637) for that, and it will require a more elaborate change.

In this change I am trying to keep them separate.  Thus, the page that generates `google/cloud/*` needs to exclude `google/cloud/bigtable`.  There is a separate target that generates the pages for  `google/cloud/bigtable/*`.

I know it is a mess, that is why I created #637.